### DISCUSSION TITLE
PyArmor get license info continues not to work 7.6.0

### DISCUSSION THREAD

**Discussion by scottwsides** [dashingsoft/pyarmor#1095] [Q&A]
I've read the old tickets on this issue and all the instructions are not working for me. I'm running 7.6.0 and I'm registering with the new .zip file method. 

<img width="989" alt="Screen Shot 2023-04-05 at 12 32 20 PM" src="https://user-images.githubusercontent.com/13578795/230173151-a2e34a62-9f5a-48d5-95b6-df674bc1510f.png">

I dont understand what the problem is. Can you please help?

**Comment by jondy** [comment]
This function is used in the obfuscated scripts, it doesn't work in plain script. It need do some extra work when using it in plain script.

Please search `get_license_info` in Pyarmor 7.x documentation, there should be examples.

**Reply by scottwsides** [reply]
Ok,  I got one of the examples to work. But the differences between the get license info examples are not clear and is confusing as to why some work and not others (presumably some difference about super-mode)
